### DISCUSSION TITLE
OpenSpace 公开课安排

### DISCUSSION THREAD

**Discussion by xilibi2003** [lbc-team/UpchainDAO#15] [General]
| 期数 | 日期 | 主题 | 分享人 | 会议/回放链接 | 
| ---- | ---- | ---- | ------ | ------------- |
|   79   |  7/3    |    Monad   |   KJ    |              |
|   80   |  7/11     |   PADO   |       |              |
|     |    7/17  |   Solana  安全 by GoPlus    |    Allen   |               |
|   81   |   7/17 (暂定)    |  zkVM by RISC0  (英文)    |      |               |
|    |  7/24 | Rooch 介绍以及开发入门 | Jolestar     |               |
|84 | 8/2|把 Solana 带到以太坊 |Ren Yu Kong/邝仁宇Growth Marketer @ Eclipse| |
|85 | 8/7或8/8 |||
|      |      |    bitcoinOS   |  廖师虎    |              |
|       |  7 / 17  之后      |   zkWasm   |   sinka   |              |
|      |      |   Do network   |     |              |

 

欢迎大家来分享，可以回复留言分享主题

**Comment by ccxdcyl** [comment]
| 期数 | 日期 | 主题 | 分享人 | 会议/回放链接 |
| ---- | ---- | ---- | ------ | ------------- |
|      |  7/24    |    Rooch 介绍以及开发入门   |   Jolestar    |              |
